dev/source-release.sh

```diff
@@ -36,7 +36,7 @@ usage () {
 }
 
 # Default repository remote name
-remote="origin"
+remote="apache"
```

Comment:
https://github.com/apache/iceberg/pull/4919/files#diff-ee187088d9f56516cc2abd9e4b4294a92d23dc09746db5b0976799131c293d2bR33

Reply:
I think this depends on where the script gets run (in the case of origin) and then how the user / box has their git remote set up.

Like mine is `upstream` vs apache, but I know most people / PMC members specifically probably use apache.

Reply:
Sorry, the link is meant to show this line above
```    echo "  -g      Specify Git remote name. Defaults to \"apache\""```

Reply:
Ahhh! Ok then absolutely yes let’s make it apache. This typically gets run manually in somebody’s machine, so `origin` is very likely incorrect (typically one’s fork).

But if it says the default is `apache`, it should be that.